src/Twitter.php

```diff
@@ -0,0 +1,335 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atymic\Twitter;
+
+use Atymic\Twitter\Exception\Request\BadRequestException;
+use Atymic\Twitter\Exception\Request\ForbiddenRequestException;
+use Atymic\Twitter\Exception\Request\NotFoundException;
+use Atymic\Twitter\Exception\Request\RateLimitedException;
+use Atymic\Twitter\Exception\Request\UnauthorizedRequestException;
+use Atymic\Twitter\Exception\RequestException as TwitterRequestException;
+use Atymic\Twitter\Traits\AccountTrait;
+use Atymic\Twitter\Traits\BlockTrait;
+use Atymic\Twitter\Traits\DirectMessageTrait;
+use Atymic\Twitter\Traits\FavoriteTrait;
+use Atymic\Twitter\Traits\FormattingHelpers;
+use Atymic\Twitter\Traits\FriendshipTrait;
+use Atymic\Twitter\Traits\GeoTrait;
+use Atymic\Twitter\Traits\HelpTrait;
+use Atymic\Twitter\Traits\ListTrait;
+use Atymic\Twitter\Traits\MediaTrait;
+use Atymic\Twitter\Traits\SearchTrait;
+use Atymic\Twitter\Traits\StatusTrait;
+use Atymic\Twitter\Traits\TrendTrait;
+use Atymic\Twitter\Traits\UserTrait;
+use GuzzleHttp\Client as HttpClient;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Subscriber\Oauth\Oauth1;
+use InvalidArgumentException;
+use Psr\Log\InvalidArgumentException as InvalidLogArgumentException;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+class Twitter
+{
+    use FormattingHelpers,
+        AccountTrait,
+        BlockTrait,
+        DirectMessageTrait,
+        FavoriteTrait,
+        FriendshipTrait,
+        GeoTrait,
+        HelpTrait,
+        ListTrait,
+        MediaTrait,
+        SearchTrait,
+        StatusTrait,
+        TrendTrait,
+        UserTrait;
+
+    public const VERSION = '3.x-dev';
+    public const RESPONSE_FORMAT_ARRAY = 'array';
+    public const RESPONSE_FORMAT_OBJECT = 'object';
+    public const RESPONSE_FORMAT_JSON = 'json';
+
+    private const DEFAULT_EXTENSION = 'json';
+    private const REQUEST_METHOD_GET = 'GET';
+    private const REQUEST_METHOD_POST = 'POST';
+    private const URL_FORMAT = 'https://%s/%s/%s.%s';
+    private const KEY_FORM_PARAMS = 'form_params';
+    private const KEY_QUERY = 'query';
+    private const KEY_FORMAT = 'format';
+
+    private const RESPONSE_CODE_400 = 400;
```

Comment:
I don't think it's worth adding these constant. It's just overhead over defining them in the switch inline and they aren't going to change.

Reply:
Usually, I prefer constants to magic numbers. In this case I guess it's fine.